authorization/src/test.rs

```diff
@@ -37,34 +37,35 @@ fn test() {
 
     // 1. set the admin
     let admin = generate_keypair();
-    set_admin::invoke(&env, &contract_id, &to_ed25519(&env, &admin));
+    let client = AuthContractClient::new(&env, contract_id);
+    client.set_admin(to_ed25519(&env, &admin));
 
     // 2. store data with user1's auth
     let user1 = generate_keypair();
-    let user1_id = &to_ed25519(&env, &user1);
+    let user1_id = to_ed25519(&env, &user1);
     let data = BigInt::from_u32(&env, 2);
 
-    let user1_nonce = nonce::invoke(&env, &contract_id, user1_id);
+    let user1_nonce = client.nonce(user1_id.clone());
```

Comment:
Let me know if you have issues with the clients taking in non-references. We could probably change the client to be references, and it was an oversight that the client didn't take references like the original invoke functions.

Reply:
I should probably change it to refs.